output.c

```diff
@@ -41,6 +42,33 @@
 #include "xwayland.h"
 #endif
 
+static void
+update_output_manager_config(struct cg_server *server)
+{
+	struct wlr_output_configuration_v1 *config = wlr_output_configuration_v1_create();
+
+	struct cg_output *output;
+	wl_list_for_each (output, &server->outputs, link) {
+		struct wlr_output *wlr_output = output->wlr_output;
+		struct wlr_output_configuration_head_v1 *config_head =
+			wlr_output_configuration_head_v1_create(config, wlr_output);
+
+		struct wlr_box *box = wlr_output_layout_get_box(server->output_layout, wlr_output);
+		if (box) {
+			config_head->state.x = box->x;
+			config_head->state.y = box->y;
+		}
+
+		// We mark the output enabled even if it is switched off by DPMS
+		config_head->state.enabled = wlr_output->current_mode != NULL && wlr_output->enabled;
```

Comment:
@Hjdskes, @emersion
I have tested the branch using Wayland backend. Do you know if we are really supposed to check `current_mode` here? If so, the Wayland output is always reported as disabled by `wlr-randr`, as no mode available for such a backend.

Commenting this test, `wlr-randr` reports the output as enabled, current mode seems to be the window size and position, transform, scale or adaptive sync parameters are also available.

Reply:
Yeah, `current_mode` is not the right thing to check for, because only the DRM backend has fixed modes.

Reply:
Thanks @emersion.
However, I have tried to check what is done in sway and observed `current_mode` is checked there since PR https://github.com/swaywm/sway/pull/5972.

Reply:
Right. What Sway is doing used to work in the past, because wlroots only supported disabling DRM outputs, and other backends refused to disable their outputs. But now that's no longer the case, and outputs which don't support modes can be disabled as well. IOW, this is a Sway bug.

We shouldn't need to touch `config_head->state.enabled` though, since we don't support DPMS. We don't need to differentiate between a disabled output and a powered off output.

Reply:
> What Sway is doing used to work in the past

And this PR is still heavily inspired by that :)